### PR TITLE
Fixed dev environment issues

### PR DESCRIPTION
**Pull request by sinisimattia**

### CODE REVIEW

**Review by SergiX44 — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -8,32 +8,33 @@
...
-      "laravel/tinker": "^2.0",
-      "laravel/ui": "^2.4",
-      "league/flysystem-aws-s3-v3": "^1.0"
+        "php": "^7.2.5|^8.0",
+        "composer/semver": "^1.5",
+        "doctrine/dbal": "^2.13",
+        "fideloper/proxy": "^4.2",
+        "fruitcake/laravel-cors": "^2.0",
+        "jenssegers/agent": "^2.6",
+        "laravel/framework": "^8.0",
```
[the remark this answers is not in the document]
Comment (sinisimattia) [reply composer.json:17]: Working on a compromise...

[the remark this answers is not in the document]
Comment (sinisimattia) [reply composer.json]: Ok... found a better solution

**Review by SergiX44 — APPROVED** [review]

**Review by jibbo — APPROVED** [review]
🤘

**Comment by jibbo** [comment]
Please, check with @SergiX44 the release procedure ;)

**Pull request merged**